hal/armv7m/pmap.c

```diff
@@ -230,12 +228,12 @@ void _pmap_init(pmap_t *pmap, void **vstart, void **vend)
 	}
 
 	ikregion = pmap_map2region(ikmap->id);
-	if (ikregion < 0) {
+	if (ikregion == 0) {
```

Comment:
`ikregion` should be declared as `unsigned int` (or `u32`) instead of `int`

Reply:
Right, fixed